src/views/DidUpgradeExplainer/DidUpgradeExplainer.test.tsx

```diff
@@ -1,18 +1,32 @@
-import { render } from '../../testing/testing';
+import { act, render } from '../../testing/testing';
+import { getPromoStatus } from '../../utilities/didUpgradePromo/didUpgradePromo';
 
 import { identitiesMock as identities } from '../../utilities/identities/IdentitiesProvider.mock';
 
 import { DidUpgradeExplainer } from './DidUpgradeExplainer';
 
+jest.mock('../../utilities/didUpgradePromo/didUpgradePromo');
+
+const getPromoStatusPromise = Promise.resolve({
+  account: '4oY2qsDpYBf2LqahCTmEC4iudf667CRT3iNoBmMLfznZoGcM',
+  remaining_dids: 1000,
+  is_active: true,
+});
+
+jest.mocked(getPromoStatus).mockReturnValue(getPromoStatusPromise);
+
 describe('DidUpgradeExplainer', () => {
-  it('should render', () => {
+  it('should render', async () => {
     const { container } = render(
       <DidUpgradeExplainer
         identity={
           identities['4tJbxxKqYRv3gDvY66BKyKzZheHEH8a27VBiMfeGX2iQrire']
         }
       />,
     );
+    await act(async () => {
+      await getPromoStatusPromise;
```

Comment:
Would it work without warnings like that? If yes, we can simplify the code above.
```suggestion
      await getPromoStatus();
```

Reply:
Nice! But even better part is that now you don’t even need a constant, and you get autocompletion for the parameter of `mockResolvedValue`. Same applies to the other file as well.